google/cloud/automl/v1beta1/automl_gapic.yaml

```diff
@@ -590,6 +769,39 @@ interfaces:
       poll_delay_multiplier: 1.5
       max_poll_delay_millis: 5000
       total_poll_timeout_millis: 300000
+    samples:
+      standalone: 
+      - region_tag: automl_vision_create_model
+        value_sets: [automl_vision_create_model]
+    
+    sample_value_sets:
+    - id: automl_vision_create_model
+      description: Demonstrates using the AutoML client to create a model.
+
+      parameters:
+        defaults:
+        - parent%project="[Google Cloud Project ID]"
+        - parent%location="us-central1"
+        - parent%dataset_id="test-dataset"
+        - model.display_name="[default_display_name_for_model]"
+        - model.image_classification_model_metadata.train_budget=64
+
+
+        attributes:
+        - parameter: parent%project
+          sample_argument_name: project_id
+          description: Your Google Cloud Project ID
+        - parameter: parent%location
+          sample_argument_name: location
+          description: Location, e.g. us-central1.
+        - parameter: model.display_name
+          sample_argument_name: model_name
+          description: the Name of the model.
+          #-------->GETTING ERROR HERE!!!!
```

Comment:
Did we figure this one out? Looks familiar.

Reply:
```/Users/munkhbayarg/apis/googleapis/artman-genfiles/java/gapic-google-cloud-automl-v1beta1/samples/src/main/java/com/google/cloud/examples/automl/v1beta1/AutomlVisionCreateModel.java:111:21: error: Syntax error, insert ":: IdentifierOrNew" to complete ReferenceExpression
/Users/munkhbayarg/apis/googleapis/artman-genfiles/java/gapic-google-cloud-automl-v1beta1/samples/src/main/java/com/google/cloud/examples/automl/v1beta1/AutomlVisionCreateModel.java:111:21: error: Syntax error, insert "AssignmentOperator Expression" to complete Assignment
/Users/munkhbayarg/apis/googleapis/artman-genfiles/java/gapic-google-cloud-automl-v1beta1/samples/src/main/java/com/google/cloud/examples/automl/v1beta1/AutomlVisionCreateModel.java:111:21: error: Syntax error, insert ";" to complete Statement

artman> Traceback (most recent call last):
  File "/artman/artman/cli/main.py", line 71, in main
    engine.run()
  File "/usr/local/lib/python3.5/dist-packages/taskflow/engines/action_engine/engine.py", line 159, in run
    for _state in self.run_iter():
  File "/usr/local/lib/python3.5/dist-packages/taskflow/engines/action_engine/engine.py", line 223, in run_iter
    failure.Failure.reraise_if_any(it)
  File "/usr/local/lib/python3.5/dist-packages/taskflow/types/failure.py", line 292, in reraise_if_any
    failures[0].reraise()
  File "/usr/local/lib/python3.5/dist-packages/taskflow/types/failure.py", line 299, in reraise
    six.reraise(*self._exc_info)
  File "/usr/local/lib/python3.5/dist-packages/six.py", line 693, in reraise
    raise value
  File "/usr/local/lib/python3.5/dist-packages/taskflow/engines/action_engine/executor.py", line 82, in _execute_task
    result = task.execute(**arguments)
  File "/artman/artman/tasks/format_tasks.py", line 39, in execute
    ['java', '-jar', path, '--replace'] + targetFiles)
  File "/artman/artman/tasks/task_base.py", line 64, in exec_command
    raise e
  File "/artman/artman/tasks/task_base.py", line 56, in exec_command
    output = subprocess.check_output(args, stderr=subprocess.STDOUT)
  File "/usr/lib/python3.5/subprocess.py", line 626, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.5/subprocess.py", line 708, in run
    output=stdout, stderr=stderr)```

Reply:
That's what I'm getting when I uncomment out commented out section

Reply:
Oh I see. This field is an `int64` type and is generated to a `long` in Java. We are not handling long types in Java yet. Fix coming soon.